### PR TITLE
feat(profiling): add profiling analytics instrumentation

### PR DESCRIPTION
**Pull request by eliashussary**
## Summary
Adds profiling analytics instrumentation to the new hovercard and slowest transactions components.

Closes https://github.com/getsentry/team-profiling/issues/176

### CODE REVIEW

File: static/app/components/profiling/functionsMiniGrid.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import {CSSProperties, Fragment} from 'react';
+import React, {CSSProperties, Fragment} from 'react';
```
Comment (Zylphrex) [inline static/app/components/profiling/functionsMiniGrid.tsx:1]: We try not to use top level `React` imports and import just what we need.

File: static/app/components/profiling/profilingTransactionHovercard.tsx [hunk 2]
```diff
@@ -104,6 +117,19 @@ export function ProfilingTransactionHovercardBody({
     });
   };
 
+  const handleLinkToFlamechartAnalyticEvent = () => {
+    trackAdvancedAnalyticsEvent('profiling_views.go_to_flamegraph', {
+      organization,
+      source: 'transaction_hovercard',
+    });
+  };
```
Comment (Zylphrex) [inline static/app/components/profiling/profilingTransactionHovercard.tsx:125]: Do we want to use different sources if they're clicking the latest vs the slowest?

Comment (eliashussary) [reply static/app/components/profiling/profilingTransactionHovercard.tsx:125]: hmmm .. i like that idea

Comment (eliashussary) [reply static/app/components/profiling/profilingTransactionHovercard.tsx]: ✅

File: static/app/components/profiling/functionsMiniGrid.tsx (hunk 1 above)
Comment (eliashussary) [reply static/app/components/profiling/functionsMiniGrid.tsx]: ✅

**Review by Zylphrex — APPROVED** [review]

**Pull request merged**